base/CMakeLists.txt

```diff
@@ -1,87 +1,82 @@
 add_subdirectory(base_tests)
 
 set(
-    HEADERS
+    SRC
     SRC_FIRST.hpp
     array_adapters.hpp
     assert.hpp
+    base.cpp
     base.hpp
     bits.hpp
     buffer_vector.hpp
     cache.hpp
     cancellable.hpp
     collection_cast.hpp
+    condition.cpp
     condition.hpp
+    condition_bada.cpp
+    condition_posix.cpp
+    condition_windows_native.cpp
     const_helper.hpp
+    deferred_task.cpp
     deferred_task.hpp
+    exception.cpp
     exception.hpp
+    gmtime.cpp
     gmtime.hpp
     limited_priority_queue.hpp
+    logging.cpp
     logging.hpp
+    lower_case.cpp
     macros.hpp
     math.hpp
     matrix.hpp
     mem_trie.hpp
     mutex.hpp
     newtype.hpp
+    normalize_unicode.cpp
+    object_tracker.cpp
     object_tracker.hpp
     observer_list.hpp
     range_iterator.hpp
     ref_counted.hpp
     rolling_hash.hpp
     scope_guard.hpp
     set_operations.hpp
+    shared_buffer_manager.cpp
     shared_buffer_manager.hpp
-    src_point.hpp stats.hpp
+    src_point.cpp
+    src_point.hpp
+    stats.hpp
     std_serialization.hpp
     stl_add.hpp
     stl_helpers.hpp
     stl_iterator.hpp
+    string_format.cpp
     string_format.hpp
+    string_utils.cpp
     string_utils.hpp
+    strings_bundle.cpp
     strings_bundle.hpp
+    sunrise_sunset.cpp
     sunrise_sunset.hpp
     swap.hpp
+    thread.cpp
     thread.hpp
+    thread_checker.cpp
     thread_checker.hpp
+    thread_pool.cpp
     thread_pool.hpp
+    threaded_container.cpp
     threaded_container.hpp
     threaded_list.hpp
     threaded_priority_queue.hpp
+    timegm.cpp
     timegm.hpp
+    timer.cpp
     timer.hpp
     worker_thread.hpp
-    internal/message.hpp
-)
-
-set(
-    SOURCES
-    base.cpp
-    condition.cpp
-    condition_bada.cpp
-    condition_posix.cpp
-    condition_windows_native.cpp
-    deferred_task.cpp
-    exception.cpp
-    gmtime.cpp
-    logging.cpp
-    lower_case.cpp
-    normalize_unicode.cpp
-    object_tracker.cpp
-    shared_buffer_manager.cpp
-    src_point.cpp
-    string_format.cpp
-    string_utils.cpp
-    strings_bundle.cpp
-    sunrise_sunset.cpp
-    thread.cpp thread_checker.cpp
-    thread_pool.cpp
-    threaded_container.cpp
-    timegm.cpp
-    timer.cpp
     internal/message.cpp
 )
 
-set(SRC ${HEADERS} ${SOURCES})
-
 omim_add_library(base "${SRC}")
```

Comment:
When i remove the double quotes, I get this: 
`-- Found Threads: TRUE
-- Configuring done
CMake Error: Cannot determine link language for target "base".
CMake Error: CMake can not determine linker language for target: base
-- Generating done
-- Build files have been written to: /Users/t.danshin/Documents/projects/omim-build-debug`